### PR TITLE
6981 - documentation for android-app-launcher, display-base64-image and parse-timestamp-to-date

### PR DESCRIPTION
**Pull request by latin-panda**
**Description**

This PR:
- Adds documentation for display-base64-image widget.
- Adds documentation for android-app-launcher widget.
- Adds documentation for parse-timestamp-to-date xpath function.

Ticket: https://github.com/medic/cht-core/issues/6981

[CHT-Core](https://github.com/medic/cht-core/pull/7248)
[CHT-Android](https://github.com/medic/cht-android/pull/199)

**License**

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by latin-panda** [comment]
Hi @garethbowen, can you please review? Thank you for your time!

**Review by garethbowen — CHANGES_REQUESTED** [review]
Very detailed documentation. I've left a couple of grammatical suggestions inline as well as a clarification about what happens in a browser.

File: content/en/apps/reference/forms/app.md [hunk 1]
```diff
@@ -107,6 +109,97 @@ Example of getting the data from the contact and assigning it to the fields necc
 | calculate | patient_uuid | Patient UUID| ||../contact/_id|
 | calculate | patient_id | Patient ID| ||../contact/patient_id|
  
+### Android App Launcher
+
+_Available in +3.13.0 and in Android device only_
+
+Use the Android App Launcher widget in an app form to configure an Android intent to launch an app installed in the mobile device, the widget will send values from input fields type `text` to the app and will assign the app's response into output fields type `text`, the only supported field type is `text`. The widget will display automatically a button to launch the app.
```
[the remark this answers is not in the document]
Comment (latin-panda) [reply content/en/apps/reference/forms/app.md]: Done!

File: content/en/apps/reference/forms/app.md [hunk 2]
```diff
@@ -107,6 +109,97 @@ Example of getting the data from the contact and assigning it to the fields necc
...
+
+To define the widget, create a `group` with the appearance `android-app-launcher`, then define the [Android intent](https://developer.android.com/reference/android/content/Intent) fields with type `text`, these fields (action, category, type, uri, packageName and flags) are optional. Every Android app have specific ways of launching with intents, check that information in the app's site and assign the corresponding values in the `default` column, see example below:
```
[the remark this answers is not in the document]
Comment (latin-panda) [reply content/en/apps/reference/forms/app.md]: Done!

File: content/en/apps/reference/forms/app.md [hunk 3]
```diff
@@ -107,6 +109,97 @@ Example of getting the data from the contact and assigning it to the fields necc
 | calculate | patient_uuid | Patient UUID| ||../contact/_id|
 | calculate | patient_id | Patient ID| ||../contact/patient_id|
  
+### Android App Launcher
+
+_Available in +3.13.0 and in Android device only_
```
[the remark this answers is not in the document]
Comment (latin-panda) [reply content/en/apps/reference/forms/app.md]: Nice, done!

**Review by garethbowen — APPROVED** [review]
Nice one!

**Pull request merged**